src/hooks/govwallet/useGovWalletBalance/useGovWalletBalance.test.tsx

```diff
@@ -131,7 +131,7 @@ describe("useGovWalletBalance", () => {
         expect(mockGetGovWalletBalance).toHaveBeenCalledTimes(1);
       });
 
-      it("should return 'INELIGIBLE' if balance is more than 10000 cents", async () => {
+      it("should return 'INELIGIBLE' if balance is more than defaultFeatures.govwalletExactBalanceValue", async () => {
```

Comment:
Nit: can change the initialization below from 10001 to refer to the variable + 1?

Reply:
updated for the + and - case